### ISSUE TITLE
Enhancement: Being able to re-view your scores

### ISSUE THREAD

**Issue by amyspark-ng** [FunkinCrew/Funkin#2441]
## What is your suggestion, and why should it be implemented?
I wish there was a way to view your scores after doing them in a song, like let's say iin freeplay, you can click your highscore to see the screen again and check how many misses or sicks or wharever you did!

Wish you could also see your scores individually when you complete a week, let's say you get the the full week score, and then you can scroll through them to see how you did individually on each song, and then you could check the highscore of that week and the thing i said earlier would apply too!

**Comment by JackXson-Real** [comment]
I agree with the first paragraph. I already made my own enchancement issue on the second one though about individual scores in weeks.

**Comment by Lasercar** [comment]
Ooooh, I could do this one too!

After all, I literally just did the https://github.com/FunkinCrew/Funkin/pull/4087 where the chart editor play(state)test goes into the results screen afterwards.

**Comment by amyspark-ng** [comment]
i guess you could make it functional but im guessing it won't be merged, if they did do this it would probably have his own keybind and option which would require art

**Comment by amyspark-ng** [comment]
yeah it would, but as i said it's art, so it's something they'd have to do themselves

**Comment by AbnormalPoof** [comment]
An idea I had was being able to see the stats for a song similar to how Geometry Dash does it:
![Image](https://github.com/user-attachments/assets/f7676587-9390-47c2-bc9f-f3a5d3c5a7b2)

**Comment by ric4455** [comment]
ive had this idea for a while, but instead of pressing a designated key youd hover on a song for a few seconds and then itd show up. heres my quick little mockup that i did just now of what i imagine it to look like
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/09bae7c4-983d-4754-89e5-95a7a0ed0c7a" />

**Comment by amyspark-ng** [comment]
this could work

**Comment by JackXson-Real** [comment]
> ive had this idea for a while, but instead of pressing a designated key youd hover on a song for a few seconds and then itd show up. heres my quick little mockup that i did just now of what i imagine it to look like <img alt="Image" width="1920" height="1080" src="https://private-user-images.githubusercontent.com/162054851/471333488-09bae7c4-983d-4754-89e5-95a7a0ed0c7a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM2ODA2MDIsIm5iZiI6MTc1MzY4MDMwMiwicGF0aCI6Ii8xNjIwNTQ4NTEvNDcxMzMzNDg4LTA5YmFlN2M0LTk4M2QtNDc1NC04OWU1LTk1YTdhMGVkMGM3YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyOFQwNTI1MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNjc3ZDM0Y2RmOGRjMGM0ZmU4MGViZWJhMzkyZGQ3NGUwYzhkNzdmNDYxNzk5ZWUzYTg0Nzg5MjE5YmY1NTIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.659DPTOLSbC6j7_0SDmu2oVHSSpjZOA2294v9ohWj1o">

Not bad
